java/ql/test/kotlin/library-tests/methods/enumClass.kt

```diff
@@ -2,3 +2,15 @@ enum class EnumClass(val v: Int) {
     enum1(1),
     enum2(1)
 }
+
+enum class EnumWithFunctions {
+
+  VAL {
```

Comment:
Is this becoming a nested class? If so, it might be worth checking if the `enclInReftype` is correctly extracted.

Reply:
I checked, it indeed gets enclosed in EnumWithFunctions